src/view.rs

```diff
@@ -22,7 +22,7 @@ use std::time::Duration;
 use regex::Regex;
 use termion::color::{Bg, Fg};
 use termion::style::Reset;
-use termion::{clear, cursor};
+use termion::cursor;
```

Comment:
the linter wants that you move this one up one line. See https://travis-ci.org/p-e-w/hegemon/jobs/653200180#L449

Reply:
shut off bot